example/v1/sphereCubeHull.js

```diff
@@ -0,0 +1,9 @@
+import { cube, sphere, writeStl, hull } from '@jsxcad/api-v1';
+
+export function main () {
+  let x = hull(sphere(),
```

Comment:
Got it. Thanks for the advice! Is that a performance thing, or a style thing?

Should I open a new pull request with that change, or would that be too small of a change for a pr?

Reply:
I doubt that performance would be affected significantly.

It's more about signaling intent.
If you use const, it indicates that nothing changes the variable.

If you always use const where possible, using let instead of const tells the reader that there's some special need for this variable to change. :)

Reply:
Ah, and don't worry about fixing it this time :)